src/Rings/mpoly-localizations.jl

```diff
@@ -382,6 +382,77 @@ function rand(S::MPolyProductOfMultSets, v1::UnitRange{Int}, v2::UnitRange{Int},
   return prod([rand(s, v1, v2, v3) for s in sets(S)])::elem_type(ambient_ring(S))
 end
 
+########################################################################
+# Localization associated to a monomial ordering                       #
+########################################################################
+
+@Markdown.doc """
+    MPolyLeadingMonOne{
```

Comment:
Personally, if I tried to find localizations at orderings in Oscar, this name would not be suggestive to me. @wdecker @jankoboehm : Any suggestions for other names?

Reply:
I agree that the name is bad, but it's the best I've got. In the end the user is supposed to use `Localization(ring, ordering)` and does not have to see this.

Reply:
Ok. I also didn't find a better name straight away. Should we receive any reasonable proposals, we can still adjust it.